DearPyGui/src/platform/Windows/mvWindowsViewport.cpp

```diff
@@ -190,6 +190,30 @@ namespace Marvel {
 		ZeroMemory(&_msg, sizeof(_msg));
 	}
 
+	void mvWindowsViewport::fullscreen()
+	{
+		size_t width = GetSystemMetrics(SM_CXSCREEN);
+		size_t height = GetSystemMetrics(SM_CYSCREEN);
+        if (mvViewport::is_fullscreened){
+			RECT rect;
+			rect.left = mvViewport::_storedX;
+			rect.top = mvViewport::_storedY;
+			rect.right = mvViewport::_storedX + mvViewport::_storedWidth;
+			rect.bottom = mvViewport::_storedY + mvViewport::_storedHeight;
+            SetWindowLongPtr(_hwnd, GWL_STYLE, WS_OVERLAPPEDWINDOW | WS_VISIBLE);
+			AdjustWindowRect(&rect, WS_OVERLAPPEDWINDOW, FALSE);
+			MoveWindow(_hwnd, mvViewport::_storedX, mvViewport::_storedY, mvViewport::_storedWidth, mvViewport::_storedHeight, TRUE);
+        } else {
+            mvViewport::_storedWidth = mvViewport::_actualWidth;
+            mvViewport::_storedHeight = mvViewport::_actualHeight;
+            mvViewport::_storedX = mvViewport::_xpos;
+            mvViewport::_storedY = mvViewport::_ypos;
+            SetWindowLongPtr(_hwnd, GWL_STYLE, WS_SYSMENU | WS_POPUP | WS_CLIPCHILDREN | WS_CLIPSIBLINGS | WS_VISIBLE);
+			MoveWindow(_hwnd, 0, 0, width, height, TRUE);
+        }
+        mvViewport::is_fullscreened = !mvViewport::is_fullscreened;
+	}
```

Comment:
I will not be able to check this tonight but I have no idea how this is compiling (or working) lol. You are treating the members as if they are static members. When you write `mvViewport::is_fullscreened`, which viewport is this referring to?  Again, I'm not sure how this is working.